src/tds/login.c

```diff
@@ -170,6 +170,22 @@ tds_set_language(TDSLOGIN * tds_login, const char *language)
 	return !!tds_dstr_copy(&tds_login->language, language);
 }
 
+bool
+tds_set_encryption(TDSLOGIN * tds_login, const char *encryption)
+{
+	if (stricmp(encryption,TDS_STR_ENCRYPTION_OFF)==0)
```

Comment:
Use `strcasecmp`, not `stricmp` for portability. Also there's a very similar function in `src/tds/config.c`, `tds_config_encryption`. You can also reach it calling `tds_parse_conf_section`.

Reply:
Sure. Can this be fixed by:
```
	case DBSETENCRYPT:
		tds_parse_conf_section(TDS_STR_ENCRYPTION, value_nonull, login->tds_login);
		return SUCCEED;
		break;
```
or maybe leave this `tds_set_encryption` and replace `stricmp` with `strcasecmp` only as you suggested?

Reply:
Calling `tds_parse_conf_section` seems fine.